### DISCUSSION TITLE
Git LFS suddenly stopped working on Windows10 machine

### DISCUSSION THREAD

**Discussion by Maxwell167** [community/community#22138] [New to GitHub]
<p>Our small company is using Git with GitHub and GitHub Desktop and LFS since over a year now. Everything worked fine. Suddenly on one machine, git lfs is not recognised by git, when i try to push.<br>
I tried reinstalling it, with no success. We didnt add any new filetypes, neither is the quota full/exeeded.<br>
Here is my log, when i try to push:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/ec47264e2620437184b846c289ee2e4732814f3a.png" data-download-href="/uploads/short-url/xId7sPUCPfuQWCcIJJQfTC8KA8G.png?dl=1" title="gitlfs output" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/4538415/181069022-e4c2c42a-7eb1-46b0-84aa-8ac62c78e60f.png" alt="gitlfs output" data-base62-sha1="xId7sPUCPfuQWCcIJJQfTC8KA8G" width="690" height="359" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/ec47264e2620437184b846c289ee2e4732814f3a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">gitlfs output</span><span class="informations">1314×685 54.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Any help is highly welcome. Thx</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi Maxwell167! :wave:</p>
<p>I’m not sure what caused the initial problem, but judging from your screenshot the large object has been committed to your regular Git history.</p>
<p>Adding LFS tracking for a file that’s already been committed to the Git repo doesn’t automatically move it to LFS. You’ll need to first remove the file completely from the repository’s regular Git history, and then add it to LFS. You can remove the file from the history by following our guide:</p>
<p><a href="https://help.github.com/en/github/authenticating-to-github/removing-sensitive-data-from-a-repository" class="onebox" target="_blank">https://help.github.com/en/github/authenticating-to-github/removing-sensitive-data-from-a-repository</a></p>
<p>Alternatively, there is a migration tool that you can use which will migrate your files over to LFS without having to delete and re-add them:</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/bozaro/git-lfs-migrate" target="_blank">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/19803303/181069023-4191f5b2-cb15-427a-bd6e-9fccddc01a44.jpeg" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/bozaro/git-lfs-migrate" target="_blank">bozaro/git-lfs-migrate</a></h3>

<p>Simple project for convert old repository for using git-lfs feature - bozaro/git-lfs-migrate</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>After that you can track and push it to LFS with these instructions:</p>
<p><a href="https://help.github.com/en/github/managing-large-files/configuring-git-large-file-storage" class="onebox" target="_blank">https://help.github.com/en/github/managing-large-files/configuring-git-large-file-storage</a></p>
